ghi/index.py

```diff
@@ -6,54 +6,27 @@
 from configuration import getConfiguration
 from github import getPool, parsePayload
 from irc import sendMessages
+from ghilogging import setup_server_logging
 from ghimastodon import sendToots
 from validation import validatePayload
 from __init__ import __version__
 
 
-def handler(event, context=None):
+def handler(event, sysd, context=None):
```

Comment:
```suggestion
def handler(event, context=None, sysd=None):
```

This needs to have `context` first because that's how AWS Lambda invokes the function. We also need to set `sysd` to None so it can still work in Lambda. This should keep it functional on both Lambda and on a server.

Reply:
Done! I wasn't able to test GHI with AWS Lambda..

Reply:
Oops,I also didn't know the handler signature was hardcoded.